packages/site-kit/components/Nav.svelte

```diff
@@ -77,14 +77,16 @@
 			<li class="hide-if-desktop" class:active={$current === ''}>
 				<a sveltekit:prefetch href="/">{home}</a>
 			</li>
-			<slot name="nav-center" />
+			<div class="nav-center">
```

Comment:
not sure if this fixes it, but I noticed that Lighthouse gives an accessibility warning for the nav, so might be worth taking a look at while this section is being updated

Reply:
because there's a `<div>` as a child of a `<ul>`? hmm. this won't fix it, no. though it won't make it worse either. should probably be addressed in a follow-up — going to merge this and release because the `npm link` workflow is killing me (seriously, npm, why do you nuke all linked packages as soon as you install something new? so dumb) but will open an issue for that